app/views/devise/registrations/_personal_details.html.erb

```diff
@@ -43,18 +46,23 @@
   <% @identifier_schemes.each do |scheme| %>
     <div class="form-group col-xs-8">
       <% if scheme.name == 'shibboleth' %>
+        <% if current_user.org.present? %>
           <label class='control-label'>
-            <span class="aria-only" aria-hidden="false"><%= scheme.description.capitalize %></span>
-            <i class="fa fa-user" title="<%= scheme.description.capitalize %>" aria-hidden="true"></i>
+            <span class="aria-only" aria-hidden="false"><%= _('Institutional credentials') %></span>
+            <i class="fa fa-user" title="<%= _('Institutional credentials') %>" aria-hidden="true"></i>
           </label>
+        <% end %>
       <% else %>
-          <%= label_tag(:scheme_name, scheme.name.capitalize, class: 'control-label') %>
+        <%= label_tag(:scheme_name, scheme.name.capitalize, class: 'control-label') %>
+      <% end %>
+
+      <% if scheme.name != 'shibboleth' || (scheme.name == 'shibboleth' && current_user.org.present?) %>
```

Comment:
Why only is required org for an user if shibboleth, if we make it mandatory for any scheme we can avoid this if.